### PR TITLE
Add Deployment ADR

### PR DESCRIPTION
**Pull request by emilysallstrom**

### CODE REVIEW

File: architectureDecisionRecords/0009HerokuDeployment.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Within [Heroku pipelines](https://devcenter.heroku.com/articles/pipelines), there are 2 options for deployments: using Heroku to automatically deploy upon merging to the main branch, or using github actions to manually deploy a pull request.
+
+For a standard Dev/Staging/Production environment set up, that leaves 2 options for deployments:
+
+1. All deployments within Heroku
+
+ * Dev environment: Heroku automatically deploys upon PR creation/update
+ * Staging environment: Heroku automatically deploys upon PR merge into main
+ * Production environment: Developer deploys code from Staging to Production by clicking the "Promote to production" button in Heroku
```
Comment (wesrisenmay-mx) [inline architectureDecisionRecords/0009HerokuDeployment.md:17]: Is it not possible to have it automatically deploy to both staging and production on merge to main?

Comment (tphalp) [reply architectureDecisionRecords/0009HerokuDeployment.md:17]: The only reason I could see to not want an automatic deploy to prod is for testing in staging. Can we have tests run in staging, and then if the tests finish successfully, that triggers a prod deployment?

Comment (tphalp) [reply architectureDecisionRecords/0009HerokuDeployment.md:17]: Actually, my comment doesn't make sense. We don't want to need to rely on having a test environment in Heroku, in addition to GH.

File: architectureDecisionRecords/0009HerokuDeployment.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+The process for every change will be:
+* Developer creates a PR within github
+* Github actions run on the PR, running all tests and linters
+* Heroku automatically deploys PR to heroku if all github actions pass
+* 2nd Developer reviews code in github and approves the PR
+* 1st Developer merges the PR
+* Heroku automatically deploys the code from main to Heroku staging environment
+* Developer verifies the app is running in staging
+* Developer promotes code from staging to production using Heroku's "Promote to production" button
+
```
Comment (tphalp) [inline architectureDecisionRecords/0009HerokuDeployment.md:67]: For now I could see testing this out with a manual deploy to prod. But, in the long run, I think automatic deployment to prod based on some trigger would be the goal.

**Review by wesrisenmay-mx — APPROVED** [review]

**Review by tphalp — APPROVED** [review]

**Pull request merged**